Tests/DependencyInjection/JMSSerializerExtensionTest.php

```diff
@@ -511,6 +512,21 @@ public function testAutoconfigureSubscribers()
         $this->assertTrue($autoconfigureInstance[EventSubscriberInterface::class]->hasTag('jms_serializer.event_subscriber'));
     }
 
+    public function testAutoconfigureHandlers()
+    {
+        $container = $this->getContainerForConfig(array());
+
+        if (!method_exists($container, 'registerForAutoconfiguration')) {
```

Comment:
In which case the `registerForAutoconfiguration` does not exists? Shouldn't this check be done also in `JMSSerializerExtension.php` ?

Reply:
It's also done in the extension (line 22)
I've blindly copied that was done :D 

I suppose it's for Symfony < 3.3 ?

Reply:
... and I just saw that `symfony-dependency-injection` require is `^3.3` so theses checks can be removed.

Reply:
The minimum dependency injection is 3.3, so probably this check and also the other you found can be removed.

Reply:
Yep